File: common/src/main/java/io/github/solusmods/eternalcore/impl/storage/network/s2c/StorageSyncPayload.java
package io.github.solusmods.eternalcore.impl.storage.network.s2c;


import net.minecraft.nbt.CompoundTag;
import net.minecraft.network.protocol.common.custom.CustomPacketPayload;

public interface StorageSyncPayload extends CustomPacketPayload {
    boolean isUpdate();

    CompoundTag storageTag();
}

File: storage-common/src/main/kotlin/io/github/solusmods/eternalcore/storage/impl/network/s2c/StorageSyncPayload.kt
package io.github.solusmods.eternalcore.storage.impl.network.s2c

import net.minecraft.nbt.CompoundTag
import net.minecraft.network.protocol.common.custom.CustomPacketPayload


interface StorageSyncPayload : CustomPacketPayload {
    val isUpdate: Boolean

    val storageTag: CompoundTag
}